trans/templates/home.html

```diff
@@ -151,8 +166,41 @@ <h3 style="color:green">
           <form
             class="form-inline"
             method="post"
-            action="{% url 'freeze_user_contest' username=user.username contest_id=contest.id%}">
+            action="{% url 'merge_print_and_freeze_contest' contest_id=contest.id%}">

```

Comment:
Got this 
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/1525580/163704043-baf4c66e-0f99-4a79-adb3-f3e1a766f733.png">

Reply:
forgot to change the form action when resolving merging issues. thanks